docs/setup/mac.md

```diff
@@ -12,20 +12,18 @@ MetaDescription: Get Visual Studio Code up and running on Mac.
 
 ## Installation
 
-1. [Download Visual Studio Code](https://go.microsoft.com/fwlink/?LinkID=534106) for Mac.
-2. Double-click on the downloaded archive to expand the contents.
-3. Drag `Visual Studio Code.app` to the `Applications` folder, making it available in the `Launchpad`.
-4. Add VS Code to your Dock by right-clicking on the icon and choosing `Options`, `Keep in Dock`.
+1. [Download Visual Studio Code for macOS](https://go.microsoft.com/fwlink/?LinkID=534106) and install it.
```

Comment:
@Rick-Anderson I did some testing on a Mac and the user will still need to manually add VS Code to the Applications folder and Dock. I think it is better to err on the side of being too explicit in the install instructions.
Thank you for calling out some of the unnecessary wording ("simply", "finally"). Our content is written by the VS Code engineers and tends to be conversational. Feel free to submit another PR if you want. I can also do a quick pass over the docs to review the use of conversational terms.